src/components/App/App.tsx

```diff
@@ -1,8 +1,31 @@
-import React from "react";
+import React, { FC } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+// import { useDispatch } from "react-redux";
+// import { useAppSelector } from "../../hooks/redux";
+// import { userSlice } from "../../store/reducers/UserSlice";
 import styles from "./App.module.scss";
+import { Header } from "../Header/Header";
+import { Home } from "../Home/Home";
+import { Footer } from "../Footer/Footer";
+import { NotFoundPage } from "../NotFoundPage";
+import { UserPage } from "../UserPage/UserPage";
+import { CountryPage } from "../CountryPage/CountryPage";
 
-const App = () => {
-  return <div className={styles.div}>Your Dream Holidays</div>;
+export const App: FC = () => {
+  return (
+    <BrowserRouter>
+      <div className={styles.wrapper}>
+        <div className={styles.forFooterWrapper}>
```

Comment:
Why do we have header and main app inside the forFooterWrapper, can we think about more descriptive name for this class?